esp/services/ws_configmgr/CMakeLists.txt

```diff
@@ -69,11 +69,14 @@ target_link_libraries(ws_configmgr
     jlib
     xmllib
     esphttp
-    environment
     dllserver
     configmgr
     )
 
+if (NOT CONTAINERIZED)
```

Comment:
can't actually see where ws_configmgr is built/included at all.. the subdir cmake file is not included in esp/services/CMakeLists.txt
 but if is, perhaps better to leave as is, and exclude it in containerized mode (as 'ws_config; already is)